### PR TITLE
Update debug2.py

### PR DESCRIPTION
**Pull request by tsabin2023**
I have fixed several lines of code and now debug2.py is outputting desired results. Have a good day.

### CODE REVIEW

File: debug2.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
 def score_to_letter_grade(grade):
-    if grade > 90:
+    if grade >= 90:
```
Comment (van-rtc) [inline debug2.py:2]: Can you make a note for each case? For Example: # Grade A: Score>=90

File: debug2.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
 def score_to_letter_grade(grade):
-    if grade > 90:
+    if grade >= 90:
         return "A"
     elif grade >= 87:
         return "B+"
-    elif grade == 80:
+    elif grade >= 81:
```
Comment (van-rtc) [inline debug2.py:6]: grade should be >=80, instead of 81, same as the rest. Please update.

**Comment by van-rtc** [comment]
Hi, I have made some comments on the codes. Can you take a look at that?

File: debug2.py (hunk 2 above)
Comment (tsabin2023) [reply debug2.py:6]: Instructions say that 81 or equal should output a B+ and this code does. Please submit another comment that code runs and all bugs are fixed.

**Comment by van-rtc** [comment]
Well done! The code ran successfully. Bugs are fixed. I will accept PR and merge with main. Project close

**Pull request merged**